tests/endpoints/test_wells4run.py

```diff
@@ -0,0 +1,57 @@
+from fastapi.testclient import TestClient
+
+from tests.fixtures.well_data import load_data4well_retrieval, load_dicts_and_users
+
+
+def test_no_run_name_url(test_client: TestClient):
+
+    response = test_client.get("/pacbio/run?run_name=xxxx")
+    assert response.status_code == 404
+
+
+def test_nonexisting_run(test_client: TestClient):
+
+    response = test_client.get("/pacbio/run/xxxx")
```

Comment:
So one of these is a filter, and the other is a resource fetch (in RESTful sense). Are these not the opposite way round, with one having a query that gives `[]` back, and the other being an entity fetch that isn't there?